hackathon_site/dashboard/frontend/src/components/general/SnackbarNotifier/SnackbarNotifier.js

```diff
@@ -27,42 +27,44 @@ export const UnconnectedSnackbarNotifier = ({
             setDisplayedSnackbars(displayedSnackbars);
         };
 
-        snackbars.forEach(({ message, options, dismissed }) => {
-            if (dismissed) {
-                closeSnackbar(options.key);
-                return;
-            }
+        if (snackbars?.length) {
```

Comment:
Was there a bug where thas was undefined somehow?

Reply:
yes, in unit tests